Feature/DESIGN 53 colour page

TrebBrennan opened the pull request:
Colour value changes have been made so that the palette passes
WCAG easier.

Some of the helper styles here (e.g. .bg-light-green) could be kept to
be used as includes on other elements?

Styles after line 453 might be better moved into the KSS and out of the ui-kit?

Commit:
DESIGN-72 work towards layout for colour page

Commit:
Syntax error fix

Commit:
Fix double comments in Colours

Commit:
Integrate updates to Colours content

Commit:
Update colour hex values

Commit:
Add updated swatch styles to Colours

Commit:
Update background colour classes

Commit:
Styles and Colour updates

Colour value changes have been made so that the palette that passes
WCAG easier.

Some of the helper styles here (e.g. .bg-light-green) could be kept to
be used as includes on other elements?

Styles after 453 should probably be moved into the KSS and out of the
ui-kit?

klepas commented:
Nice.

I think @petronbot is best suited to the do the refactoring as she has been working on the KSS templates and styles recently — is that OK Petra?

Commit:
Edits from @joolswood

Commit:
Change to $grey value

$grey changed to #6e6e6e so that it passes AA on $non-white (see
breadcrumbs current) :tada:

Commit:
Colour change now passes AA

Commit:
Fix spelling & linting errors

Commit:
Move Guide styles out of UI Kit and into Guide
